mobsfscan/rules/patterns/ios/swift/swift_rules.yaml

```diff
@@ -70,9 +70,12 @@
 - id: ios_biometric_bool
   message: Biometric authentication should be based on Keychain, not based on bool.
   input_case: exact
-  pattern: \.evaluatePolicy\(.deviceOwnerAuthentication
+  pattern:
+    - \.evaluatePolicy\(.deviceOwnerAuthentication
```

Comment:
Can these two be replaced with  `\.evaluatePolicy\(.*\.deviceOwnerAuthentication` ?

Reply:
yes, that's cleaner, will update shortly.